v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,22 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT sub.question_id, avg(sub.mean_score_per_user_per_question) as mean_score_per_question, sub.number, corr(sub.mean_score_per_user_per_question, sub2.average_assessment_score_for_user) as discrimination
+FROM (
+    select user_id, question_id, assessment_questions.number, avg(instance_questions.points/assessment_questions.max_points) as mean_score_per_user_per_question
```

Comment:
For some reason, the score_perc seems to be 0 in a lot of places where points are not 0.